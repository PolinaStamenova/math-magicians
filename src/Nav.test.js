import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router, Link } from 'react-router-dom';

it('renders correctly', () => {
  const tree = renderer
    .create(
      <Router>
        <Link to="/">Home</Link>
      </Router>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
