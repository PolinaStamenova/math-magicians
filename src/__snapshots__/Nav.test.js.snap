// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<a
  href="/"
  onClick={[Function]}
>
  Home
</a>
`;
